src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,35 @@
 package mate.jdbc;
 
+import java.util.List;
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.dao.ManufacturerDaoImpl;
+import mate.jdbc.models.Manufacturer;
+
 public class Main {
     public static void main(String[] args) {
+        ManufacturerDao manufacturerDao = new ManufacturerDaoImpl();
+
+        Manufacturer.Builder manufacturerBuilder = new Manufacturer.Builder();
+        manufacturerDao
+                .create(manufacturerBuilder.setName("Mercedes").setCountry("Germany").build());
+        manufacturerDao
+                .create(manufacturerBuilder.setName("BMW").setCountry("Germany").build());
```

Comment:
```suggestion
        Manufacturer.Builder germanManufacturerBuilder = new Manufacturer.Builder().setCountry("Germany");
        manufacturerDao
                .create(germanManufacturerBuilder .setName("Mercedes").build());
        manufacturerDao
                .create(germanManufacturerBuilder .setName("BMW").build());
```
If you decided to use builder - use its advantages